.github/workflows/publish.yml

```diff
@@ -24,6 +24,23 @@ jobs:
         env:
           REF: ${{ github.ref }}
         run: |
+          # we are using ubuntu-latest as our riscv64 cross build machine
+          if [ "${{ matrix.os }}" = ubuntu-latest ]; then
```

Comment:
I am confused. Why not just install binfmt and qemu-user-static in all use cases?

Reply:
Because we absolutely do NOT want to do a cross build in cases where we can avoid it (we have a hosted ARM runner for a reason -- otherwise the build can't even complete within the time GH allows for a build VM to exist). In fact, this problem can bite us down the road on riscv64 as well -- but given that it is only subset of packages for now -- we seem to be doing fine.

Reply:
Installing `binfmt` doesn't force you to use it; it only provides the option. Install it always, use it when you want to.

The bigger issue here is that we seem to be using `[ "${{ matrix.os }}" = ubuntu-latest ]` as a proxy for "I am building for riscv". That is bound to create confusion (and likely errors) further down the line. If we are building for riscv, let's be explicit about it.